package.json

```diff
@@ -82,6 +82,7 @@
     "class-validator": "^0.12.2",
     "concurrently": "^5.3.0",
     "cors": "^2.8.5",
+    "cz": "^1.8.2",
```

Comment:
Is this package needed?

Reply:
Ah, I get it. You most likely did not have the package installed locally to run the 'npm run commit' script. Go ahead and update the 'npm run commit' script to use npx which allows you to run the script without package installation. 

'npx git-cz'